### PR TITLE
Fix tests after deploying LPTs via Proxy

### PR DESCRIPTION
**Pull request by tensojka**

### CODE REVIEW

**Comment by tensojka** [comment]
@Chepelau Please feel free to squash and merge this once the tests are green

File: tests/test_run_integration_tests.cairo [hunk 1]
```diff
@@ -133,12 +133,6 @@ func test_non_eth_round_trip{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, ran
     return ();
 }
 
-@external
-func test_deposit_liquidity{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(){
```
Comment (Chepelau) [inline tests/test_run_integration_tests.cairo:137]: why is this whole function just deleted

Comment (tensojka) [reply tests/test_run_integration_tests.cairo:137]: because the test doesn't actually do anything (see the deleted file)

Comment (Chepelau) [reply tests/test_run_integration_tests.cairo:137]: i don't see a connection to test_deposit func, that is a legitimate test

**Pull request merged**